manifests/init.pp

```diff
@@ -49,6 +49,9 @@
         }
       } elsif $::operatingsystem == 'Darwin' and $::macosx_productversion_major =~ '10\.[9,10,11]' {
         $_package_file_name = "${puppet_agent::package_name}-${puppet_agent::params::master_agent_version}-1.osx${$::macosx_productversion_major}.dmg"
+      } elsif $::operatingsystem == 'aix' and $::architecture =~ 'PowerPC_POWER[5,6,7]' {
+        $aix_ver_number = regsubst($::architecture,'PowerPC_POWER(\d+)$','\1')
+        $_package_file_name = "${puppet_agent::package_name}-${puppet_agent::params::master_agent_version}-1.aix${aix_ver_number}.1.ppc.rpm"
```

Comment:
AIX numbering is not always x.1 -- 5.3, 6.1, 7.1.

Which version where you testing?